impeller/renderer/context.h

```diff
@@ -52,6 +52,10 @@ class Context {
     kVulkan,
   };
 
+  /// The maximum number of tasks that should ever be stored for
+  /// `StoreTaskForGPU`.
+  static constexpr int32_t kMaxTasksAwaitingGPU = 10;
```

Comment:
Oh by the way.  I'm open to increasing this.  I just pulled this number out of nowhere but I knew I didn't want it to be completely unbounded.

Reply:
I would leave a note that this was arbitrarily chosen so we're not afraid to increase it in the future.

Reply:
done!